drivers/soc/atmel/soc.c

```diff
@@ -69,6 +69,12 @@ static const struct at91_soc __initconst socs[] = {
 #endif
 #ifdef CONFIG_SOC_SAM9X60
 	AT91_SOC(SAM9X60_CIDR_MATCH, SAM9X60_EXID_MATCH, "sam9x60", "sam9x60"),
+	AT91_SOC(SAM9X60_CIDR_MATCH, SAM9X60D5M_EXID_MATCH,
+		 "sam9x60d6m", "sam9x60"),
```

Comment:
Take example on sama5d2 SiP lines to have a coherent display between the different SiP products.

Reply:
Hi @noglitch, I changed the descriptions as You suggested. I hope it is okay now.